### ISSUE TITLE
Add to navigation guidelines

### ISSUE THREAD

**Issue by mmenestr** [patternfly/patternfly-org#2077]
Add a section to say - when to use tertiary navs vs tabs?

**Comment by mmenestr** [comment]
On hold- waiting on Gina investigation?

**Comment by mmenestr** [comment]
Closing because duplicate of this one: https://app.zenhub.com/workspaces/design-doc-workspace-5f08c64d6a6ef2001d56a458/issues/patternfly/patternfly-org/2443
